pkg/cvo/status.go

```diff
@@ -327,6 +327,14 @@ func updateClusterVersionStatus(cvStatus *configv1.ClusterVersionStatus, status
 		failingCondition.Reason = failingReason
 		failingCondition.Message = failingMessage
 	}
+	if failure != nil &&
+		skipFailure &&
+		progressReason == "ClusterOperatorUpdating" &&
+		strings.Contains(progressMessage, "longer than expected") {
```

Comment:
:trollface: 

This looks a bit brittle (someone changes a message and it breaks `oc adm upgrade status`). I think we'll need something more robust. I would not be opposed to `convertErrorToProgressing` adding a `Slow::` prefix or similar to the reason it returns, and we would key on that. Or something similar.

Reply:
Done.

Reply:
The updated versions still adds the `slowCOUpdatePrefix` prefix to the message, but I think we should use communicate this way through `Reason` (which is somewhat expected to be machine-readable, unlike message)

Reply:
> I would not be opposed to convertErrorToProgressing adding a Slow:: prefix or similar to the reason it returns

I did not get your intension in the last round. Let me do it.

Reply:
Done.